_pages/en_US/dumping-games-and-carts.txt

```diff
@@ -0,0 +1,142 @@
+---
+title: "Dumping Titles and Game Cartridges"
+---
+
+{% include toc title="Table of Contents" %}
+
+For support (in English) with GodMode9, as well as help with scripting and to get updates and info, join [GodMode9 on Discord](https://discord.gg/BRcbvtFxX4).
+{: .notice--primary}
+
+### Required Reading
+
+Aside from creating and restoring NAND backups, GodMode9 has the functionality to dump installed titles to an installable `.cia` file, dump game cartridge data to a `.3ds` ROM file, or directly install a game cartridge to the system.
```

Comment:
"dump game cartridge data" -> "to dump game cartridge data"; "or directly install" -> "(or/and) to directly install"